static/js/admin/src/CreateContainerForm.js

```diff
@@ -17,6 +17,7 @@ define('CreateContainerForm', [ 'jquery', 'jquery-ui', 'underscore', 'RemoteStat
 	CreateContainerForm.prototype.open = function(resultObject) {
 		this.options.containerType = this.getContainerType(resultObject);
 		this.options.title = 'Create ' + this.options.containerType;
+		this.options.privateOnlyBox = this.options.containerType !== "AdminUnit";
```

Comment:
lets make this `staffOnly` or `staffOnlyBox` for the sake of using consistent language.